sos/plugins/kubernetes.py

```diff
@@ -9,13 +9,29 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.plugins import Plugin, RedHatPlugin
+from sos.plugins import Plugin, RedHatPlugin, UbuntuPlugin
 from os import path
 
 
-class kubernetes(Plugin, RedHatPlugin):
+class Ubuntukubernetes(Plugin, UbuntuPlugin):
 
-    """Kubernetes plugin
+    """Ubuntu Kubernetes plugin
+    """
+
+    files = ("/snap/bin/kubectl",)
+
+    def setup(self):
+        self.add_cmd_output([
+            "snap run kubectl get nodes -o wide",
+            "snap run kubectl cluster-info",
```

Comment:
This is really the only new command being captured by `Ubuntukubernetes`, the other bits are all already captured. Is there any reason we couldn't add `cluster-info` to the existing collection and just set `kube_cmd` to `snap run kubectl` for Ubuntu while keeping it `kubectl` elsewhere?

Reply:
Sure, I'll have a look at it and re-submit a PR later.